test/XRP/issued-currency-client-integration.test.ts

```diff
@@ -223,33 +223,29 @@ describe('IssuedCurrencyClient Integration Tests', function (): void {
     // WHEN a transaction is sent to the account without a destination tag
     const xrpClient = new XrpClient(rippledGrpcUrl, XrplNetwork.Test)
     const xrpAmount = '100'
-    const transactionHash = await xrpClient.send(
+    const transactionResult = await xrpClient.sendXrp(
       xrpAmount,
       wallet.getAddress(),
       wallet2,
     )
-    const transactionStatus = await xrpClient.getPaymentStatus(transactionHash)
 
     // THEN the transaction fails.
-    assert.exists(transactionHash)
-    assert.equal(transactionStatus, TransactionStatus.Failed)
+    assert.exists(transactionResult.hash)
+    assert.equal(transactionResult.status, TransactionStatus.ClaimedCostOnly)
```

Comment:
Using the actual `TransactionResult`, I noticed that the original source of the transaction status (`getPaymentStatus`) was not creating a granular enough distinction from the raw server transaction code (like `'tecPATH_DRY` for example).  These tests are now correct but I've made a task to update our logic around converting from raw string code to a `TransactionStatus` so that we can make sure that the core logic exists only once and is reused wherever this conversion is happening: https://app.asana.com/0/1190819649053561/1199159961044989/f